packages/s/sudo-rs/package.yml

```diff
@@ -0,0 +1,23 @@
+name       : sudo-rs
+version    : 0.2.1
+release    : 1
+source     :
+    - https://github.com/memorysafety/sudo-rs/archive/refs/tags/v0.2.1.tar.gz : 67469b5402375cd6f2d76d00bbce170238e2b7be9ff83bc1de88d14535288a38
+homepage   : https://github.com/memorysafety/sudo-rs
+license    :
+    - Apache-2.0
+    - MIT
+component  : system.base
```

Comment:
This should be in something different until we're ready to commit to it being in `system.base`